### PR TITLE
Config guide files fix markdown format.

### PR DESCRIPTION
**Pull request by chirag-wagento**
## This PR is a:
- [ ] New topic
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement
 

## Summary
Fixes Config guide files typo.
 
When this pull request is merged, it will...

### CODE REVIEW

File: guides/v2.1/config-guide/bootstrap/magento-bootstrap.md [hunk 1]
```diff
@@ -14,20 +14,20 @@ functional_areas:
...
-*	Create an instance of <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/Bootstrap.php" target="_blank">\Magento\Framework\App\Bootstrap</a>. <!-- It requires initialization parameters to be specified in constructor.  Normally, the $_SERVER super-global variable is supposed to be passed there. -->
-*	Create a Magento application instance (<a href="{{ site.mage2000url }}lib/internal/Magento/Framework/AppInterface.php" target="_blank">\Magento\Framework\AppInterface</a>).
+*	Include [app/bootstrap.php]({{ site.mage2000url }}app/bootstrap.php){:target="&#95;blank"} which performs essential initialization routines, such as error handling, initializing the autoloader, setting profiling options, setting the default timezone, and so on.
+*	Create an instance of [\\Magento\\Framework\\App\\Bootstrap]({{ site.mage2000url }}lib/internal/Magento/Framework/App/Bootstrap.php){:target="&#95;blank"}. <!-- It requires initialization parameters to be specified in constructor.  Normally, the $_SERVER super-global variable is supposed to be passed there. -->
```
Comment (shrielenee) [inline guides/v2.1/config-guide/bootstrap/magento-bootstrap.md]: Remove the double slashes. Also please ensure they don't exist anywhere else in your changes.

File: guides/v2.1/config-guide/bootstrap/magento-bootstrap.md [hunk 2]
```diff
@@ -57,28 +55,28 @@ $app = $bootstrap->createApplication('Magento\Framework\App\Http');
...
-*	<a href="#config-boot-entry-http">HTTP entry point</a>
-*	<a href="#config-boot-entry-static">Static resource entry point</a>
-*	<a href="#config-boot-entry-media">Media resource entry point</a>
+*	[HTTP entry point](#config-boot-entry-http)
+*	[Static resource entry point](#config-boot-entry-static)
+*	[Media resource entry point](#config-boot-entry-media)
 
-<h3 id="config-boot-entry-http">HTTP entry point</h3>
-<a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/Http" target="_blank">\Magento\Framework\App\Http</a> operates as follows:
+### HTTP entry point {#config-boot-entry-http}
```
Comment (shrielenee) [inline guides/v2.1/config-guide/bootstrap/magento-bootstrap.md]: Double slashes.

File: guides/v2.1/config-guide/bootstrap/magento-bootstrap.md [hunk 3]
```diff
@@ -87,30 +85,28 @@ We have the following entry point applications (that is, applications defined by
 	5.	If there is any other application initialization error, display a user-friendly "Page Not Found" page with HTTP status code 404 (Not Found).
 	6.	On any other error, display a user-friendly "Service Unavailable" page with HTTP response 503 and generate an error report and display its ID on the page.
 
-<h3 id="config-boot-entry-static">Static resource entry point</h3>
-<a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/StaticResource.php" target="_blank">\Magento\Framework\App\StaticResource</a> is an application for retrieving static resources (for example, CSS, JavaScript, and images). It postpones any actions with a static resource until the resource is requested.
+### Static resource entry point {#config-boot-entry-static}
```
Comment (shrielenee) [inline guides/v2.1/config-guide/bootstrap/magento-bootstrap.md]: Double slashes.

File: guides/v2.1/config-guide/bootstrap/magento-bootstrap.md [hunk 4]
```diff
@@ -87,30 +85,28 @@ We have the following entry point applications (that is, applications defined by
...
 
 	All future requests for the static resource are processed by the server the same as static files; that is, without involving the entry point. If it's necessary to synchronize published files with original ones, the `pub/static` directory should be removed; as a result, files are automatically republished with the next request.
 
-<h3 id="config-boot-entry-media">Media resource entry point</h3>
-<a href="{{ site.mage2000url }}app/code/Magento/MediaStorage/App/Media.php" target="_blank">Magento\MediaStorage\App\Media</a> retrieves media resources (that is, any files uploaded to media storage) from the database. It is used whenever the database is configured as a {% glossarytooltip d95142d7-023f-451c-a2e9-dd88763dcd70 %}media storage{% endglossarytooltip %}.
+### Media resource entry point {#config-boot-entry-media}
+[Magento\\MediaStorage\\App\\Media]({{ site.mage2000url }}app/code/Magento/MediaStorage/App/Media.php){:target="&#95;blank"} retrieves media resources (that is, any files uploaded to media storage) from the database. It is used whenever the database is configured as a {% glossarytooltip d95142d7-023f-451c-a2e9-dd88763dcd70 %}media storage{% endglossarytooltip %}.
```
Comment (shrielenee) [inline guides/v2.1/config-guide/bootstrap/magento-bootstrap.md]: Double slashes.

File: guides/v2.1/config-guide/bootstrap/magento-how-to-set.md [hunk 5]
```diff
@@ -40,46 +40,42 @@ The following table discusses the bootstrap parameters you can set:
...
 </table>
 
-<div class="bs-callout bs-callout-info" id="info">
-<span class="glyphicon-class">
-  <ul><li>Not all bootstrap parameters are documented at this time.</li>
-  	<li>You now set the Magento mode (developer, default, production) using the <a href="{{ page.baseurl }}/config-guide/cli/config-cli-subcommands-mode.html"><code>magento deploy:mode:set {mode}</code></a> command.</li></ul></span>
-</div>
+{:.bs-callout .bs-callout-info}
+*   Not all bootstrap parameters are documented at this time.
+*   You now set the Magento mode (developer, default, production) using the \[\`magento deploy:mode:set {mode}\`\]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-mode.html) command.
```
Comment (shrielenee) [inline guides/v2.1/config-guide/bootstrap/magento-how-to-set.md]: Check this URL - the slashes etc. don't see right.

File: guides/v2.1/config-guide/bootstrap/magento-how-to-set.md [hunk 6]
```diff
@@ -103,18 +99,18 @@ To set a variable:
...
 
-*	<a href="#mode-specify-ubuntu">Specify a bootstrap variable for Apache on Ubuntu</a>
-*	<a href="#mode-specify-centos">Specify a bootstrap variable for Apache on CentOS</a>
+*	[Specify a bootstrap variable for Apache on Ubuntu](#mode-specify-ubuntu)
+*	[Specify a bootstrap variable for Apache on CentOS](#mode-specify-centos)
 
-<h4 id="mode-specify-ubuntu">Specify a bootstrap variable for Apache on Ubuntu</h4>
-This section assumes you've already set up your virtual host. If you have not, consult a resource such as <a href="https://www.digitalocean.com/community/tutorials/how-to-set-up-apache-virtual-hosts-on-ubuntu-14-04-lts" target="_blank">this digitalocean tutorial</a>.
+#### Specify a bootstrap variable for Apache on Ubuntu {#mode-specify-ubuntu}
+This section assumes you've already set up your virtual host. If you have not, consult a resource such as [this digitalocean tutorial](https://www.digitalocean.com/community/tutorials/how-to-set-up-apache-virtual-hosts-on-ubuntu-14-04-lts){:target="&#95;blank"}.
```
Comment (shrielenee) [inline guides/v2.1/config-guide/bootstrap/magento-how-to-set.md]: Should be "DigitalOcean".

File: guides/v2.1/config-guide/bootstrap/magento-how-to-set.md [hunk 7]
```diff
@@ -147,8 +143,8 @@ To set a Magento bootstrap variable using your web server's environment:
 	*	Ubuntu: `service apache2 restart`
 	*	CentOS: `service httpd restart`
 
-<h4 id="mode-specify-centos">Specify a bootstrap variable for Apache on CentOS</h4>
-This section assumes you've already set up your virtual host. If you have not, consult a resource such as <a href="https://www.digitalocean.com/community/tutorials/how-to-set-up-apache-virtual-hosts-on-centos-6" target="_blank">this digitalocean tutorial</a>.
+#### Specify a bootstrap variable for Apache on CentOS {#mode-specify-centos}
+This section assumes you've already set up your virtual host. If you have not, consult a resource such as [this digitalocean tutorial](https://www.digitalocean.com/community/tutorials/how-to-set-up-apache-virtual-hosts-on-centos-6){:target="&#95;blank"}.
```
Comment (shrielenee) [inline guides/v2.1/config-guide/bootstrap/magento-how-to-set.md]: Should be "DigitalOcean".

**Comment by chirag-wagento** [comment]
Hi @shrielenee I have removed Double slashes and fix text . Please check

Thanks

**Pull request merged**